cmd/atlas/atlas.go

```diff
@@ -53,6 +62,74 @@ func initConfig() {
 	}
 }
 
+// createConfigFromMongoCLIConfig creates atlasCLI config file from mongoCLI config file.
+func createConfigFromMongoCLIConfig(err error) bool {
+	// search mongoCLI config only if atlasCLI config doesn't exist
+	var e viper.ConfigFileNotFoundError
+	if !errors.As(err, &e) {
+		return false
+	}
+
+	mongoCLIConfigPath := config.ReadMongoCLIConfig()
+	if mongoCLIConfigPath == "" {
+		return false
+	}
+
+	atlasCLIConfigPath := copyConfig(mongoCLIConfigPath)
+	if atlasCLIConfigPath == "" {
+		return false
+	}
+
+	_, _ = fmt.Fprintf(os.Stderr, `we have used %s to generate %s
+
+`, mongoCLIConfigPath, atlasCLIConfigPath)
+
+	return true
+}
+
+// copyConfig copies config in oldConfigPath to the correct config location.
+func copyConfig(oldConfigPath string) string {
```

Comment:
why is this method swallowing all the errors?

Reply:
if we get an error in copying the mongocli config we don't print it but we follows the "regular" flow: if the error in `config.Load()` is `ConfigFileNotFoundError`, go-ahead. Otherwise, print the error